src/commands.py

```diff
@@ -11,8 +11,9 @@
 from blocks import BlockedUsers
 from client import client
 from config import LogTypes, CMD_PREFIX, MAILBOX
-from forwarder import MessageForwarder
+from forwarder import message_forwarder
```

Comment:
Do we use this in this file?

Reply:
Yeah, it's for the user to reply to for the `$reply` command.

That's a bit different from all the other code which fetches a user - e.g. it works for discord replies too.

Could likely unify it all into one, but I'll leave that for a future change.